index.html

```diff
@@ -63,5 +63,12 @@ <h1 class="large l-align-center l-mb-mega">&lt;gaia-button&gt;</h1>
     <gaia-button circular><i data-icon="reload"></i></gaia-button>
     <gaia-button circular><i data-icon="airplane"></i></gaia-button>
   </section>
+
+  <hr/>
+
+  <section>
```

Comment:
I don't think we need this section since gaia-button doesn't define press down style. But we may add `onclick` event handler at other cases to have keyboard feature.